cms/articles/models.py

```diff
@@ -93,13 +95,16 @@ def previous_releases(self, request: "HttpRequest") -> "TemplateResponse":
         return response
 
 
+class StatisticalArtivlePageAdminForm(PageWithHeadlineFiguresAdminForm, PageWithCorrectionsAdminForm): ...
```

Comment:
I would argue that `PageWithHeadlineFiguresAdminForm` is only relevant to statistical articles, so should live in `cms.articles.forms` as `StatisticalArticlePageAdminForm` (not Arti`c`le, not Arti`v`le)

Reply:
Thanks for spotting the typo. Relocated as suggested in [bd42a4c](https://github.com/ONSdigital/dis-wagtail/pull/152/commits/bd42a4ceb77b92b7bb4aea7fd98069baf55ca7bc).

Reply:
excellent. also move this definition to `cms/articles/forms.py`